### PR TITLE
message_admin - Fix quirk when using "Create Draft"

### PR DESCRIPTION
**Pull request by totten**
Overview
----------------------------------------

In `message_admin` extension, there is a quirk when creating a new draft of a translated message.  This fixes it.

Steps to reproduce
----------------------------------------

(*Steps performed in Firefox 92*)

* Enable `message_admin`
* Navigate to "Admin => Comm => Message Templates => System Workflow Messages"
* Pick a template and add a new "Translation"
* In this new translation, click "Create draft"
* Edit the subject line (eg add new text at the end - or double-click a word and delete it)

Before
----------------------------------------

As soon as you start typing (eg ~300ms later), the content resets. Your initial changes are applied off-screen to the "Current" revision, and the screen redraws to show the (previous/unedited) "Draft" revision. 

(After that happens, editing is normal. You can flip back/forth between tabs+fields as expected.)

After
----------------------------------------

Works from the get-go.

Technical Details
----------------------------------------

The problem seems to involve the manner in which changes propagate between the Angular parts (ie `MsgtpluiEdit` <=> `crmMsgadmEditContent` <=> `crmMosaico`). (1) I hacked `copyTranslations()` (`dest[fld] = 'copy: ' + src[fld]`) -- propagating this extra data-change caused things to work intuitively. (2) While hacking around in other ways, I sometimes noticed warnings about non-assignability that touched on the `options` for `crmMsgadmEditContent`.

It seems to involve a quirk around how the bidirectional `bindings:{options:'='}` works. The patch decomposes `options` so that it doesn't lean as heavily on deep+bidir `options`.

Comments
----------------------------------------

The original problem is a bit quirky and doesn't always reproduce. I *think* that it makes a difference that you're working on a draft of a *new* translation. But I haven' strictly determined that. However, it was quite reproducible while I was developing this patch. It may not be worthwhile for a reviewer to reproduce the original problem. The main question would be whether the change seems regressive in any way.

### CODE REVIEW

File: ang/crmMonaco.js [hunk 1]
```diff
@@ -65,6 +65,12 @@
             // FIXME: else: retry?
           };
 
+          if ($attr.ngDisabled) {
```
Comment (eileenmcnaughton) [inline ang/crmMonaco.js:68]: @colemanw I think this change to crmMonaco is the only part of this pr that needs review (the rest is internal to the extension & gets a pass under 'alpha/hidden' level ) 

do you have any thoughts on it?

**Comment by eileenmcnaughton** [comment]
[message] =&gt; DB Error: no database selected

**Comment by eileenmcnaughton** [comment]
test this please

File: ang/crmMonaco.js (hunk 1 above)
Comment (totten) [reply ang/crmMonaco.js:68]: FWIW, there are similar techniques in `ui-bootstrap-tpls.js` which inspired this.

Comment (eileenmcnaughton) [reply ang/crmMonaco.js:68]: @totten ok - is getting this merged holding you up? I am not familiar with that code but I guess if it's a pre-existing pattern....

Comment (eileenmcnaughton) [reply ang/crmMonaco.js:68]: I guess my feeling is that if you feel confident in that change then go ahead & merge this

**Comment by colemanw** [comment]
I'm good with this change

**Pull request merged**